### DISCUSSION TITLE
Clarify clarity-core or ng-clarity

### DISCUSSION THREAD

**Discussion by mathisloge** [vmware-clarity/ng-clarity#477] [General]
Hey, 
I'm a bit confused which is now the correct way to go. 
I don't want to end up using something, which is deprecated in a few weeks/months.

https://github.com/vmware-clarity/ng-clarity/discussions/402#discussioncomment-4351191
and
https://github.com/vmware-clarity/ng-clarity/discussions/402#discussioncomment-4223283

has left me with the question of which version of clarity to use. Could you clarify your plans?

**Comment by Jinnie** [comment]
Clarity Angular is the flagship product at the moment. We view Core as a future goal, but it has to still undergo some more evaluation before we can give more detail.
CC: @pdaigle

**Reply by squidjam** [reply]
Quick question... If that's the case, where can one subscribe to get the latest on this?
I'm actually building something using Core and loving it a lot (since it can be used outside of only Angular) and I would like to be on top of this info :)

Cheers.
